src/Helpers/logout.js

```diff
@@ -0,0 +1,5 @@
+const Logout = () => {
+  sessionStorage.clear();
```

Comment:
I think since we are saving the details on localStorage, we do not need to clear sessionStorage

Reply:
Changed